test/common/stats/thread_local_store_test.cc

```diff
@@ -474,6 +475,60 @@ TEST_F(StatsThreadLocalStoreTest, HistogramScopeOverlap) {
   tls_.shutdownThread();
 }
 
+TEST_F(StatsThreadLocalStoreTest, ForEach) {
```

Comment:
Can you add a TODO for me to add histograms to this when for each becomes available for them?